backend/database/seeders/UserSeeder.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\User;
+use Illuminate\Database\Seeder;
+
+class UserSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        User::factory()
+            ->hasUserDetail()
+            ->hasRoles(1, [
+                'name' => 'customer',
+                'description' => 'Customer'
+            ])
```

Comment:
Medyo naconfuse lang ako sa part na to. Kasi sa pagkakaintindi ko and base sa docs. When you pass a second paramenter on factory magic method it override the value. So yung 1 kasi admin base doon sa factory mo sa Roles. o mali lang din ako ng pagkaintindi dito. Btw, lets discuss nalang din on this.